credential-controller.js

```diff
@@ -159,6 +159,9 @@ function factory(
         if(!('identity' in session) && self.credential.sysIsPublic) {
           // display a public credential to a non-authenticated user
           _display('credentialInfo');
+        } else if('CREDENTIAL_ADMIN' in session.identity.sysPermissionTable) {
```

Comment:
@dlongley note that on L165 it is also assumed that session.identity exists. https://github.com/digitalbazaar/bedrock-angular-credential/blob/adminPermission/credential-controller.js#L165

Would you like to add a check for that prior to this entire subroutine?